pkg/repository/repository.go

```diff
@@ -202,9 +206,9 @@ func (r FFRepository) Close() {
 }
 
 func formatFlagKey(identifier string) string {
-	return "flags/" + identifier
+	return "flag/" + identifier
```

Comment:
Do these key changes work for the existing default cache in the go sdk? i.e. is this `formatFlagKey` function used for both getting and setting these keys or are there any updates needed elsewhere to enable this?

Reply:
Used for both getting and setting - this makes it consistent with what we use for the 'dtokey' too 

```
// KeyFeature identifies flag messages from ff server or stream
	KeyFeature = "flag"
```

```
// KeySegment identifies segment messages from ff server or stream
	KeySegment = "target-segment"
```